dev-qt/qtwaylandscanner/qtwaylandscanner-5.15.3.9999.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+QT5_MODULE="qtwayland"
+inherit qt5-build
+
+DESCRIPTION="Tool that generates certain boilerplate C++ code from Wayland protocol xml spec"
+
+if [[ ${QT5_BUILD_TYPE} == release ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ppc ~ppc64 ~riscv ~sparc ~x86"
+fi
+
+SLOT=5
```

Comment:
I suppose this will move to the eclass later? (related to #225)

Reply:
Would make sense yes.